env/board.py

```diff
@@ -1,5 +1,16 @@
-from .player import Player, Card
-from typing import List, Optional
+from .player import Player, PlayerState
+from .card import Card
+from .actions import (
+    Action,
+    PlayCardAction,
+    BetAction,
+    RevealCardAction,
+    LoseCardAction,
+    PassAction,
+    parse_notation,
+)
+from typing import List, Optional, Union
```

Comment:
If we use python 3.10 or above we can use the builtin ˋlist` for typing 
And we can also replace Union with 
ObjA | ObjB